src/main/resources/sql_scripts/001-create-base_tables.sql

```diff
@@ -77,8 +77,8 @@ CREATE TABLE IF NOT EXISTS shift_to_user (
 -- Table: user
 CREATE TABLE IF NOT EXISTS "user" (
                                             id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-                                            firstname VARCHAR(50) NOT NULL,
-                                            lastname VARCHAR(50) NOT NULL,
+                                            first_name VARCHAR(50) NOT NULL,
+                                            last_name VARCHAR(50) NOT NULL,
```

Comment:
to jest stworzona baza danych. To jest stworzona tabela. To jest na produkcji (nie ma, ale po to właśnie mamy liquibase'a), aby zachowywać się, jakbyśmy mieli.

Należy:
1. Nie zmieniać żadnych historycznych plików SQL
2. "ucieczka do przodu" - zmiana tabeli:

```
ALTER TABLE "user"
RENAME COLUMN firstname TO first_name;

ALTER TABLE "user"
RENAME COLUMN lastname TO last_name;
```

Reply:
i tak już od teraz zawsze, wszędzie. Nie edytujemy żadnych plików, które zostały wykonane. Dodajemy nowe, które zmieniają istniejącą strukturę bd.